docs/snippets/all/archetypes/image_send_columns.rs

```diff
@@ -23,7 +23,13 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     // Log the ImageFormat and indicator once, as static.
     let format = rerun::components::ImageFormat::rgb8([width as _, height as _]);
-    rec.log_static("images", &[&format as _, &rerun::Image::indicator() as _])?;
+    rec.log_static(
+        "images",
+        &[
+            &format as &dyn rerun::ComponentBatch,
+            &rerun::Image::indicator(),
+        ],
+    )?;
```

Comment:
Inference only breaks due to the explicit indicator shenanigans here, which I think is fine since they're going away.
Without the indicator, this would now come down to this:
```rust
rec.log_static("images", &format)?;
```
which is pretty neat.